devops/nightly.yml

```diff
@@ -34,5 +33,9 @@ jobs:
     pyVersions: [3.6, 3.7] 
 
 - template: notebook-job-template.yml
+  parameters:
+    vmImage: 'ubuntu-16.04'
+
+- template: perf-tests-job-template.yml
```

Comment:
Should we have a separate "nightly performance" build?

Reply:
I thought nightly contains everything that runs nightly? :D What do you see as the benefits of separating it?

Reply:
It keeps the current Nightly build focussed on correctness. Performance would be separate from that

Reply:
OK! Separated it out